<ui version="4.0" >
 <class>CSurfaceViewComp</class>
 <widget class="QWidget" name="CSurfaceViewComp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="SurfacePlotFrame" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_2" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="_3" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="OrthoCheck" >
           <property name="text" >
            <string>Ortho</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="colorlegend" >
           <property name="text" >
            <string>Legend</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="AutoScaleCheck" >
           <property name="text" >
            <string>Autoscale</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="mouseinput" >
           <property name="text" >
            <string>Mouse</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="shader" >
           <property name="text" >
            <string>Shading</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="_4" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Normals&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_5" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QSlider" name="normalsquality" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum" >
              <number>3</number>
             </property>
             <property name="maximum" >
              <number>32</number>
             </property>
             <property name="pageStep" >
              <number>4</number>
             </property>
             <property name="value" >
              <number>3</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="invertedAppearance" >
              <bool>true</bool>
             </property>
             <property name="tickPosition" >
              <enum>QSlider::TicksAbove</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="normalslength" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum" >
              <number>1</number>
             </property>
             <property name="maximum" >
              <number>100</number>
             </property>
             <property name="pageStep" >
              <number>5</number>
             </property>
             <property name="value" >
              <number>8</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="invertedAppearance" >
              <bool>true</bool>
             </property>
             <property name="tickPosition" >
              <enum>QSlider::TicksAbove</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_7" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="normButton" >
       <property name="text" >
        <string>Std</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lighting" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Lighting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="lightingswitch" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_8" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Polygon Offset&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="offsSlider" >
         <property name="maximum" >
          <number>30</number>
         </property>
         <property name="pageStep" >
          <number>5</number>
         </property>
         <property name="value" >
          <number>5</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition" >
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval" >
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_9" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Resolution&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="ResolutionSlider" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>10</number>
         </property>
         <property name="pageStep" >
          <number>1</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition" >
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval" >
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
